docs_user/modules/openstack-pull_openstack_configuration.adoc

```diff
@@ -86,99 +86,3 @@ Once it's done, you should have into your local path a directory per services su
       ▾ glance/
```

Comment:
Perhaps later we can also rename this to something like "Compare control plane configurations" to make a further distinction between the current "get configuration" and "pull configuration". Not something that should block this PR though. LGTM, thanks!

Reply:
Ah this is actually related to one of the merge conflicts. @klgill already renamed it to "Reviewing the openstack control plane configuration", +1 on that name. So we just have to extract this piece back to its original place in the flow: https://openstack-k8s-operators.github.io/data-plane-adoption/user/#proc_retrieving-services-topology-specific-configuration_reviewing-configuration

Reply:
Yeah @jistr, the following preview shows the current structure of the "Reviewing the OpenStack control plane configuration": https://openstack-k8s-operators.github.io/data-plane-adoption/user/#reviewing-the-openstack-control-plane-configuration_adopting-bare-metal-provisioning

Are we okay with the structure and placement in that preview?

Reply:
For one half of that content yes, the other half needs to move to the beginning of the control plane adoption, like Mathieu has it in this PR.

Reply:
@jistr @matbu  OK FYI: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/414